src/widgetastic_patternfly4/tabs.py

```diff
@@ -0,0 +1,49 @@
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import View
+
+
+class Tab(View):
+    """Represents the Patternfly Tab widget.
+
+    Selects itself automatically when any child widget gets accessed, ensuring that the widget is
+    visible.
+
+    http://patternfly-react.surge.sh/patternfly-4/components/tabs
+    """
+    # The text on the tab. Can be omitted if it is the same as the tab class name capitalized
+    TAB_NAME = None
+
+    # Locator of the Tab selector
+    TAB_LOCATOR = ParametrizedLocator(
+        './/div[contains(@class, "pf-c-tabs")]/ul/li[button[text()={@tab_name|quote}]]')
```

Comment:
**REQUIRED**
Please use `normalize-space(.)` instead of `text()` in xpath. It prevents cases when the tab name contains leading or trailing whitespaces.